homeassistant/components/roborock/__init__.py

```diff
@@ -110,6 +110,11 @@ async def setup_device(
     # Verify we can communicate locally - if we can't, switch to cloud api
     await coordinator.verify_api()
     coordinator.api.is_available = True
+    try:
+        await coordinator.get_maps()
```

Comment:
_Was browsing through recent changes to see why Roborock takes 40s to set up._

Any reason we would call `get_maps` before the first config_entry_refresh? 

And why not put this in the normal coordinator but just not fetch it every time (if a slower interval is what we want)